content/source/docs/enterprise/api/policy-sets.html.md

```diff
@@ -86,6 +92,35 @@ Key path                               | Type            | Default | Description
 }
 ```
 
+### Sample Payload with VCS repository
+
+```json
+{
+  "data": {
+    "attributes": {
+      "name": "production",
+      "description": "This set contains policies that should be checked on all production infrastructure workspaces.",
+      "global": false,
+      "policies-path": "/policy-sets/foo",
+      "vcs-repo": {
+        "branch": "master",
+        "identifier": "hashicorp/my-policy-sets",
+        "ingress-submodules": false,
+        "oauth-token-id": "ot-7Fr9d83jWsi8u23A"
+      }
+    },
+    "relationships": {
+      "workspaces": {
+        "data": [
+          { "id": "ws-2HRvNs49EWPjDqT1", "type": "workspaces" }
+        ]
+      }
+    },
+    "type": "policy-sets"
+  }
+}
+```
+
```

Comment:
Consider inverting this? "Sample Payload", and then take the existing one and put it below it as "Sample Payload (Deprecated)" or similar.

Reply:
Same for the other examples.